docs/migrations/chain-upgrade-guide-043.md

```diff
@@ -0,0 +1,233 @@
+<!--
+order: 1
+-->
+
+# Chain Upgrade Guide to v0.43
+
+This document provides guidelines for a chain upgrade from v0.42 to v0.43 and an example of the upgrade process using `simapp`. {synopsis}
+
+::: tip
+You must upgrade to Stargate v0.42 before upgrading to v0.43. If you have not done so, please see [Chain Upgrade Guide to v0.42](/v0.42/migrations/chain-upgrade-guide-040.html).
+:::
+
+## Prerequisite Readings
+
+- [Upgrading Modules](../building-modules/upgrade.html) {prereq}
+- [In-Place Store Migrations](../core/upgrade.html) {prereq}
+- [Cosmovisor](../run-node/cosmovisor.html) {prereq}
+
+Cosmos SDK v0.43 introduces a new way of handling chain upgrades that no longer requires exporting state to JSON, making the necesssary changes, and then creating a new chain with the modified JSON as the new genesis file.
+
+Instead of starting a new chain, the upgrade binary will read the existing database and perform in-place store migrations. This new way of handling chain upgrades can be used alongside [Cosmovisor](../run-node/cosmovisor.html) to make the upgrade process seemless.
```

Comment:
```suggestion
Instead of starting a new chain, the upgrade binary will read the existing database and perform in-place store migrations. This new way of handling chain upgrades can be used alongside [Cosmovisor](../run-node/cosmovisor.html) to make the upgrade process seamless.
```